src/resolve/resolved-bus.c

```diff
@@ -1600,7 +1600,7 @@ static int bus_property_get_ntas(
         return sd_bus_message_close_container(reply);
 }
 
-static BUS_DEFINE_PROPERTY_GET_ENUM(bus_property_get_dns_stub_listener_mode, dns_stub_listener_mode, DnsStubListenerMode);
+static BUS_DEFINE_PROPERTY_GET_ENUM(bus_property_get_dns_stub_listener_mode, dns_stub_listener_mode, DNSStubListenerMode);
```

Comment:
hmm, we use the lowercase spelling for pretty much *all* structs/enums so far, DNSStubListenerExtra is the one outlier. That's why in #16985 i wen the other way, and fixes that one to use lower case spelling

Reply:
i.e. compare DnsPacket, DnsResourceRecord, DnsStream …

Reply:
Hmm, I asked @ssahani to use uppercase in another PR. I guess consistent "Dns" spelling is OK too.